COLLABORATOR_GUIDE.md

```diff
@@ -93,12 +93,9 @@ to land but is [author ready](#author-ready-pull-requests), add the
 
 ## Accepting Modifications
 
-All modifications to the Node.js code and documentation should be performed via
-GitHub pull requests, including modifications by Collaborators and TSC members.
-A pull request must be reviewed, and must also be tested with CI, before being
-landed into the codebase. There may be exceptions to the latter (the changed
-code cannot be tested with a CI or similar). If that is the case, please leave a
-comment that explains why the PR does not require a CI run.
+All modifications to Node.js start as GitHub pull requests. This includes
+modifications by Collaborators and TSC members. A pull request must pass code
```

Comment:
I can live with this, but for the record, I was trying to avoid using _include_ twice in a short sentence like that. While TSC members are Collaborators, I don't know that the fact of that overlap is important here. I also considered removing the sentence entirely because it's redundant. "All modifications start as X. This includes modifications proposed by Y and Z." Well, yeah, you *just* said **all** modifications. That includes...**all** modifications.

And that's enough rambling for me. Accepting the suggestion now. :-D